reflekt-plugin/src/main/kotlin/org/jetbrains/reflekt/plugin/generation/ir/SmartReflektIrTransformer.kt

```diff
@@ -42,7 +42,7 @@ class SmartReflektIrTransformer(
     private val messageCollector: MessageCollector? = null,
 ) : BaseReflektIrTransformer(messageCollector) {
     private val importChecker = ImportChecker(classpath)
-    private val sources = HashMap<String, SourceFile>()
+    private val sources: HashMap<String, SourceFile> = HashMap()
```

Comment:
Usually, when the type of a collection property is specified explicitly, a more abstract type is used:
```suggestion
    private val sources: MutableMap<String, SourceFile> = HashMap()
```